traffic_portal/app/src/common/modules/form/server/form.server.tpl.html

```diff
@@ -112,11 +112,11 @@
 
             <label for="profile" ng-class="{'has-error': hasError(serverForm.profile)}">Profile *</label>
             <div ng-class="{'has-error': hasError(serverForm.profile), 'has-feedback': hasError(serverForm.profile)}">
-                <select id="profile" name="profile" class="form-control" ng-model="server.profileId" ng-options="profile.id as profile.name for profile in profiles" required>
+                <select id="profile" name="profile" class="form-control" ng-model="server.profileName" ng-options="profile.name as profile.name for profile in profiles" required>
```

Comment:
we no longer need `as profile.name` here, since we're no longer valuing and labeling by different properties.

Reply:
we do.. it didn't work when `as profile.name` was removed.